### PR TITLE
Computer science recipe

### PR DESCRIPTION
**Pull request by KarinaMendez2**
for: https://app.zenhub.com/workspaces/ce-styles-5cdc84baee941c07853fbbf2/issues/openstax/kitchen/391

### CODE REVIEW

**Review by LostMojo — APPROVED** [review]

File: books/computer-science/bake [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    bake_exercises: true
+  )
+
+  BakeUnclassifiedNotes.v1(book: book)
+
+  book.chapters.each do |chapter|
+    BakeLearningObjectives.v2(chapter: chapter, add_title: false)
+    BakeNonIntroductionPages.v1(chapter: chapter)
+    BakeChapterGlossary.v1(chapter: chapter, metadata_source: metadata)
+    BakeChapterSummary.v1(chapter: chapter, metadata_source: metadata, klass: 'section-summary')
```
Comment (wilkus27) [inline books/computer-science/bake:50]: I think we are using `MoveCustomSectionToEocContainer` for section summary now :thinking: That's what I see in the newest recipes for other books. For example in Anthropology.

File: books/computer-science/shorten [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+require 'bundler/inline'
+
+gemfile do
+  gem 'openstax_kitchen', ENV['USE_LOCAL_KITCHEN'] ? { path: '/code/kitchen' } : '5.0.0'
```
Comment (wilkus27) [inline books/computer-science/shorten:8]: Should we update kitchen version here?

File: books/computer-science/shorten [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  gem 'slop', '4.8.2'
+  gem 'byebug'
+end
+
+recipe = Kitchen::BookRecipe.new(book_short_name: :computer_science) do |doc|
```
Comment (wilkus27) [inline books/computer-science/shorten:13]: I think in `shorten` script we are using different name then in `bake`. It should be `:computer_science_short` or something like that.

File: books/finance/shorten [hunk 4]
```diff
@@ -10,7 +10,7 @@ gemfile do
   gem 'byebug'
 end
 
-recipe = Kitchen::BookRecipe.new(book_short_name: :microbiology_short) do |doc|
+recipe = Kitchen::BookRecipe.new(book_short_name: :finance) do |doc|
```
Comment (wilkus27) [inline books/finance/shorten:13]: Similar here. Shouldn't be `:finance_short`?

**Review by wilkus27 — CHANGES_REQUESTED** [review]

File: books/computer-science/bake (hunk 1 above)
Comment (KarinaMendez2) [reply books/computer-science/bake:50]: 🤔 should `BakeChapterSummary` be deprecated?

Comment (wilkus27) [reply books/computer-science/bake:50]: Good question :thinking: I would say yes... But I'm not sure. I think Kendra can say more bout it. She created `MoveCustomSectionToEocContainer` and I think the goal was to deprecate all modules for specific sections.

Comment (KarinaMendez2) [reply books/computer-science/bake:50]: Awesome, thanks for the clarification K!

**Review by wilkus27 — APPROVED** [review]

**Pull request merged**